Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN pip install --upgrade pip wheel jmespath-terminal
 # openssh - included for ssh-keygen
 # ca-certificates 
 # wget - required for installing jp
-RUN apk update && apk add bash gcc make openssl-dev libffi-dev musl-dev jq openssh ca-certificates wget openssl git && update-ca-certificates
+RUN apk update && apk add bash gcc make openssl-dev libffi-dev musl-dev jq openssh ca-certificates wget openssl git docker && update-ca-certificates
```

Comment:
What scenario is this for? The user could run `apk add docker` if they need docker.
It adds close to 100MB to our Docker image.
https://pkgs.alpinelinux.org/package/v3.6/community/aarch64/docker
356MB -> 433MB